build/Dockerfile.jupyter

```diff
@@ -1,11 +1,16 @@
 ARG BASE_IMAGE=base-charmed-spark:latest
+ARG JUPYTERLAB_VERSION=4.0.0
 FROM $BASE_IMAGE
 
+ARG JUPYTERLAB_VERSION
```

Comment:
nit: `JUPYTERLAB_VERSION` defined twice (in line 2 and 5)?

Reply:
I believe it is actually required! See [here](https://stackoverflow.com/a/56748289) for more information.